util/templates/documentation_modbus.tpl

```diff
@@ -14,7 +14,6 @@ modbus: rs485tcpip
 id: {{ .id }}
 host: {{ .host }} # Hostname
 port: {{ .port }} # Port
-rtu: true
```

Comment:
Ich hatte das mit 'configure' abgeglichen. Dort wird auch ein rtu in die Konfiguration geschrieben. Wenn das nicht nötig ist müssten wir das auch ich configure selbst entfernen.

Reply:
Good catch. Das ist massiv merkwürdig. Dann unterscheiden sich `rs485tcpip` und `tcpip` nämlich nicht mehr. Ich fürchte wir müssen uns die `configure` Logik nochmal anschauen.

Reply:
Ja, ich steck da fachlich nicht tief genug drin. Ich gehe auch davon aus, dass das in Bug in `configure` ist und deine Änderung korrekt.

Wenn ich mir das `modbus.tpl` anschaue ist der einzige Unterschied zwischen `rs485tcpip` und `tcpip`, dass ersterer für evcc das `rtu` schreibt. Der `rtu` Parameter im Template hat da auch keinen Effekt drauf.

Reply:
Das muss bleiben!!!